### ISSUE TITLE
missing type definition for qs

### ISSUE THREAD

**Issue by fxnm** [tighten/ziggy#789]
### Ziggy version

v2.4.0

### Laravel version

11.31.0

## Problem
When trying to build a react + typescript frontend with ziggy, the build fails, with the following error message:

```
Error: vendor/tightenco/ziggy/src/js/index.d.ts(1,26): error TS7016: Could not find a declaration file for module 'qs'. '/node_modules/qs/lib/index.js' implicitly has an 'any' type.
  Try `npm i --save-dev @types/qs` if it exists or add a new declaration (.d.ts) file containing `declare module 'qs';`
Error: Process completed with exit code 2.
```

## Expectation
The library ships with the type definitions for `qs`

### Ziggy call and context

```js
-
```

### Ziggy configuration

```js
-
```

### Route definition

```php
-
```

**Comment by leecolarelli-tgb** [comment]
Any timeline on this? Currently preventing my build from deploying to Laravel Vapor, and wondering if i should roll back or wait. Thank you.

**Comment by jrmajor** [comment]
@leecolarelli-tgb As a workaround, try `npm i -D @types/qs`. It suppressed all warnings for me.

**Comment by leecolarelli-tgb** [comment]
> @leecolarelli-tgb As a workaround, try `npm i -D @types/qs`. It suppressed all warnings for me.

Thank you very much @jrmajor. I have implemented this and the app builds successfully.

**Comment by bakerkretzmar** [comment]
Fixed in v2.4.1.
